mongo/integration/unified_spec_test.go

```diff
@@ -36,13 +36,23 @@ import (
 )
 
 const (
-	gridFSFiles       = "fs.files"
-	gridFSChunks      = "fs.chunks"
-	cseMaxVersionTest = "operation fails with maxWireVersion < 8"
+	gridFSFiles  = "fs.files"
+	gridFSChunks = "fs.chunks"
 )
 
 var (
 	defaultHeartbeatInterval = 50 * time.Millisecond
+	skippedTestDescriptions  = map[string]struct{}{
+		// SPEC-1403: This test checks to see if the correct error is thrown when auto encrypting with a server < 4.2.
+		// Currently, the test will fail because a server < 4.2 wouldn't have mongocryptd, so Client construction
+		// would fail with a mongocryptd spawn error.
+		"operation fails with maxWireVersion < 8": {},
+		// GODRIVER-2123: The two tests below use a failpoint and a socket or server selection timeout.
+		// The timeout causes the eventual clearing of the failpoint in the test runner to fail with an
+		// i/o timeout.
+		"Ignore network timeout error on find":             {},
+		"Network error on minPoolSize background creation": {},
+	}
```

Comment:
See GODRIVER-2123. As discussed offline, some SDAM spec tests setup a failpoint _and_ set either a socket or server selection timeout on the client. Since we currently use the same client for administrative ops (like failpoint teardown) and regular test operations, failpoint teardown sometimes times out for these two tests.

I've added a `skippedTestDescriptions`  map to the `mongo/integration/unified_spec_test` runner just like the one in `mongo/integration/unified/unified_spec_runner`.

Reply:
Awesome! Thank you for investigating GODRIVER-2123 and adding the `cseMaxVersionTest` to this `skippedTestDescriptions` slice. I like this pattern, since it makes it explicit why we are skipping a test, and leaves a greppable comment for future developers working on those tickets.